webpack.config.js

```diff
@@ -1,25 +1,26 @@
 require( 'es6-promise' ).polyfill();
 
-var path = require( 'path' );
-var webpack = require( 'webpack' );
-var fs = require('fs');
-var NODE_ENV = process.env.NODE_ENV || 'development';
-var ExtractTextPlugin = require( 'extract-text-webpack-plugin' );
+const path = require( 'path' );
+const webpack = require( 'webpack' );
+const NODE_ENV = process.env.NODE_ENV || 'development';
+const ExtractTextPlugin = require( 'extract-text-webpack-plugin' );
 
-var IS_HOT_UPDATE = ( process.env.NODE_ENV !== 'production' );
```

Comment:
`IS_HOT_UPDATE` seemed safe to remove. It was added in https://github.com/Automattic/jetpack/commit/533d98c142dbcd71054aff505c7fe2128851a364#diff-11e9f7f953edc64ba14b0cc350ae7b9dR9 and anything relying on it has since been removed.